src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java

```diff
@@ -122,32 +122,32 @@ private void testActions() {
 
         ______TS("Typical case: no picture");
 
-        profilePage.editProfileThroughUi("", "short.name", "e@email.tmt", "inst", "Usual Nationality",
+        profilePage.editProfileThroughUi("", "short.name", "e@email.tmt", "inst", "Singaporean",
                                          "male", "this is enough!$%&*</>");
-        profilePage.ensureProfileContains("short.name", "e@email.tmt", "inst", "Usual Nationality",
+        profilePage.ensureProfileContains("short.name", "e@email.tmt", "inst", "Singaporean",
                                           "male", "this is enough!$%&*</>");
         profilePage.verifyStatus(Const.StatusMessages.STUDENT_PROFILE_EDITED);
-        
+
```

Comment:
Also, if the database consists of student with invalid nationalities (e.g "Singaprean", with a typo), are they properly handled? After this fix is submitted it should not happen but prior it might happen (as you can see in the test data).